notebooks/community/feature_store/sdk-feature-store-exporting_client.ipynb

```diff
@@ -0,0 +1,1954 @@
+{
```

Comment:
<p>Break this cell into 4 cells. Move Dataset after Objective.</p><p> </p><p>Add link to BQ pricing.</p><p> </p><p>Cloud BigQuery -&gt; BigQuery</p><p>Colab -&gt; notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>